DOL.WHD.Section14c.Test/Business/ErrorLogsControllerTest.cs

```diff
@@ -62,6 +62,19 @@ public void ErrorLog_ReturnsAllLogs()
         }
 
         [TestMethod]
+        [ExpectedException(typeof(ApiDataException),
+            "Log not found.")]
+        public void ErrorLog_ReturnsAllLogs_Invalid()
+        {
+            // Arrange
+            var service = new ErrorLogsController(_errorLogRepository);
+            ((ErrorLogRepositoryMock)_errorLogRepository).AddShouldFail = true;
```

Comment:
I set the default value to false in Mock.
public bool AddShouldFail { get; set; } = false;

Reply:
Right, but I meant there's not a `[TestInitialize]` where the mock would be recreated or `AddShouldFail` would be reset.  So once you set it to `true`, it stays `true` for all subsequent tests.  Coincidentally it works out fine for now because it is supposed to be `true` for all the subsequent tests, but if someone ever added a new test where it was supposed to be `false`, it could throw them for a loop.